pages/getting-started.mdx

```diff
@@ -17,36 +17,66 @@ We're using a monorepo to manage our libraries and apps. Each library can be ins
 Our libraries in the libs folder are structured to be consumed by our apps as well as used by external developers using our tooling.
 
 **Utility Libraries**
-The [Common](../sdk/utilities/common-utilities/overview) and [Contract](../sdk/utilities/contract-utilities/overview) libraries provide foundational utility that can be composed and integrated into applications.
+These are libraries that are used to provide foundational utility that can be composed and integrated into applications.
 
-- [DAO Data SDK](../sdk/data/overview)
-- [Component Library](../sdk/ui/overview)
+- DAO Data SDK
+- Component Library
 
 **Feature Libraries**
 These compose together other libraries such as the DAO Data SDK and the Component Library to create "smart components" that can be integrated into applications.
 
-- [DAOhaus Connect](../sdk/features/daohaus-connect)
-- [Form Builder](../sdk/features/form-builder)
-- [Transaction Builder](../sdk/features/transaction-builder)
+- DAOhaus Connect
+- Tx Builder
+- Form Builder
 
 Each of these libraries can be found in our [DAOhaus v3 Monorepo](https://github.com/HausDAO/daohaus-monorepo) in the `/libs` folder and are included in the Toolbox documentation.
 
 ## DAOhaus v3 Monorepo Overview
 
-Our monorepo follows the [recommended Nx structure](https://nx.dev/more-concepts/folder-structure#integrated-repo-folder-structure) with `apps` and `libs` folders. The `apps` folder contains our core applications, subgraphs, and infrastructure jobs. If you want to see how we utilize the libraries in the _DAOhaus Toolbox_ in our own v3 app development check out the packages in this folder.
+Our monorepo follows the [recommended Nx structure](https://nx.dev/structure/applications-and-libraries) with `apps` and `libs` folders. The `apps` folder contains our core applications, subgraphs, and infrastructure jobs. If you want to see how we utilize the libraries in the _DAOhaus Toolbox_ in our own v3 app development check out the packages in this folder.
```

Comment:
https://nx.dev/structure/applications-and-libraries is broken

Reply:
I'll remove the link -- that must have been removed recently